includes/Embed_Block.php

```diff
@@ -155,8 +155,10 @@ protected function render_block_html( array $attributes ) {
 		$player_style = sprintf( 'width: %dpx; height: %dpx; margin: %s', absint( $attributes['width'] ), absint( $attributes['height'] ), esc_attr( $margin ) );
 		$poster_style = ! empty( $poster ) ? sprintf( '--story-player-poster: url(%s)', $poster ) : '';
 
-		wp_enqueue_style( 'amp-story-player' );
-		wp_enqueue_script( 'amp-story-player' );
+		if ( ! function_exists( 'is_amp_endpoint' ) || ! is_amp_endpoint() ) { // @todo This will need to be changed when the plugin includes an is_amp_endpoint() polyfill.
```

Comment:
This may need to be revised when the Web Stories plugin includes:

```
if ( ! is_amp_endpoint() ) {
    function is_amp_endpoint() {
        return is_singular( 'web-story' );
    }
}
```

Then again, I suppose not because this script will never get enqueued on a singular `web-story`.